git_machete/git_operations.py

```diff
@@ -754,11 +754,15 @@ def merge_fast_forward_only(self, branch: LocalBranchShortName) -> None:  # refs
 
     def rebase(self, onto: AnyRevision, fork_revision: AnyRevision, branch: LocalBranchShortName, opt_no_interactive_rebase: bool) -> None:
         def do_rebase() -> None:
+            # Let's use `OPTS` suffix for consistency with git's built-in env var `GIT_DIFF_OPTS`
+            git_machete_rebase_opts_var = 'GIT_MACHETE_REBASE_OPTS'
+            rebase_opts_str = os.environ.get(git_machete_rebase_opts_var)
+            rebase_opts = filter(None, rebase_opts_str.split(' ')) if rebase_opts_str else []
             try:
                 if opt_no_interactive_rebase:
-                    self._run_git("rebase", "--onto", onto, fork_revision, branch)
+                    self._run_git("rebase", *rebase_opts, "--onto", onto, fork_revision, branch)
                 else:
-                    self._run_git("rebase", "--interactive", "--onto", onto, fork_revision, branch)
+                    self._run_git("rebase", *rebase_opts, "--interactive", "--onto", onto, fork_revision, branch)
```

Comment:
Since we have `rebase_opts`, we could do:
```python
if opt_no_interactive_rebase:
    rebase_opts.append("--interactive")

self._run_git("rebase", *rebase_opts, "--onto", onto, fork_revision, branch)
```

Reply:
It's `if NOT opt_no_interactive_rebase: rebase_opts.append("--interactive")` ;) changed